src/templates/landing_pages/graduate_program.jsx

```diff
@@ -169,7 +169,7 @@ const GraduateProgram = () => {
                                 <div className='gr-7 gr-12-m'>
                                     <h2>{('Quants')}</h2>
                                     <div className='box fill-bg-color'>
-                                        <p>{('Manage and optimise pricing and risk management software for exotic options, Forex, and CFDs.')}</p>
+                                        <p>{('Manage and optimise pricing and risk management software for exotic options, forex, and CFDs.')}</p>
```

Comment:
i saw most of the text are {('text here')}, isnt it supposed to be {it.L('text here')}?

Reply:
@mustofa-binary no, we don't translate this page as it's only meant for english